### PR TITLE
Tech/clenup redesign

### PR DESCRIPTION
**Pull request by ala-n**
# PR Details

- marquee reworked
- structure cleanup
- partial color fixes
- typography updated
- fix sidebar paddings after header update

### CODE REVIEW

File: pages/src/esl-media-demo/test-media.less [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    height: 100%;
+
+    z-index: 1;
+  }
+  .demo-content {
+    z-index: 2;
+
+    &.contrast {
+      color: white;
+      text-shadow: 0 0 4px rgba(0,0,0,.8);
```
Comment (dshovchko) [inline pages/src/esl-media-demo/test-media.less:36]:
```suggestion
      text-shadow: 0 0 4px rgba(0, 0, 0, .8);
```

File: pages/src/landing/landing.less [hunk 2]
```diff
@@ -0,0 +1,4 @@
+@import "./marquee/marquee.less";
+@import "./benefits/benefits.less";
+@import "./browser-support/browser-support.less";
+//@import "./overview/overview.less";
```
Comment (dshovchko) [inline pages/src/landing/landing.less:4]: extra comment

**Review by dshovchko — APPROVED** [review]

**Review by abarmina — APPROVED** [review]

File: pages/src/landing/landing.less (hunk 2 above)
Comment (ala-n) [reply pages/src/landing/landing.less:4]: Comment left by purpose. Commented import references component that will not be presented in the upcoming first wave of redesign but will be in the future

**Pull request merged**